packages/@guardian/source-foundations/src/utils/resets.ts

```diff
@@ -17,6 +17,7 @@ const fieldset = `
 `;
 
 // remove styling of invalid input elements that gets applied in Firefox
+// this uses the SCSS parent selector and in the future we should look to make this just CSS
```

Comment:
If you move to JSDoc here, this will make it clearer to consumers that this is Sass and not CSS!

Reply:
if input is not valid CSS, we should probably rename it to make that clear.

given we export `resets.resetCSS`, maybe something like `resets.inputSCSS`? i don't think it would be clear (or expected) that we'd export scss from this package (it's news to me!)

Reply:
agreed 👍

Reply:
Could we make this valid CSS by simply doing:

```css
input:invalid {
  box-shadow: none;
}
```

Reply:
<img width="429" alt="Screenshot 2022-09-22 at 17 04 20" src="https://user-images.githubusercontent.com/1771189/191796803-37642552-086f-478f-80d2-bc1c44a1a777.png">

it's individually applied to the `TextArea` (and `TextInput`) in Source.. not sure about if it's used by any of our consumers..

I suppose that would be fine to move into our main reset as long as we're sure we'd like this behaviour everywhere?

Reply:
If it is only used twice… is is useful to have this as an abstraction? Should we simply inline it in both of these styles?

Reply:
@SiAdcock do you know why this is not in the global reset already? is there a good reason for it?

Reply:
let leave this for now, renaming should clear up confusion and we can work on an improvement as a separate change